src/util/apparmor.h

```diff
@@ -6,5 +6,29 @@
 
 #include <stdlib.h>
 
+typedef struct NameSet NameSet;
+
+enum {
+        _APPARMOR_E_SUCCESS,
+
+        APPARMOR_E_DENIED,
```

Comment:
Can you prefix these with `BUS_APPARMOR_*`? The reason for the `BUS_*` prefix in selinux was that we do not want to conflict with selinux names, which sometimes start with `selinux_*`. I think the same is warranted for apparmor for consistency, even if the library uses `aa_*` as prefix (otherwise we should really drop it from the functions as well).